test/cancel-stuck-txs.test.js

```diff
@@ -0,0 +1,22 @@
+import assert from 'node:assert'
+import {
+  getMessagesInMempool,
+  getRecentSendMessage
+} from '../lib/cancel-stuck-txs.js'
+
+describe('cancel stuck transactions', () => {
+  describe('getMessagesInMempool(addr)', () => {
+    it('should return messages in mempool', async () => {
+      const messages = await getMessagesInMempool(
+        '0x000000000000000000000000000000000000dEaD'
```

Comment:
Can we use a real address, please? For example, the current f4 address that is used by the spark-evaluate service. We can hard-code this address in this file, no need to create a wallet.

Reply:
We could, but it most of the time won't have any messages in the mempool. What difference does it make?

Reply:
I don't see any transactions for 0x000000000000000000000000000000000000dEaD, maybe this address is handled in a special way?

https://filfox.info/en/address/0x000000000000000000000000000000000000dEaD

How about using the spark-publish address? That one should have one transaction in the mempool quite often, or not?

Reply:
Well, but what would it test? Are you proposing that if one message is returned we check the shape of it? I don't yet understand what would change if we changed the address to one that more likely has messages in the mempool

Reply:
Yeah, it would be ideal to test the shape of the returned message. OTOH, if the response is empty most of the time, then I agree this does not matter that much.

Reply:
I think we can only have a message there reliably if we switch to testnet and create a new tx as part of the test